content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -140,111 +140,121 @@ spec:
-  }
-  ```
-  -->
-  ```gRPC
-  service DevicePlugin {
-        // GetDevicePluginOptions 返回与设备管理器沟通的选项。
-        rpc GetDevicePluginOptions(Empty) returns (DevicePluginOptions) {}
-
-        // ListAndWatch 返回 Device 列表构成的数据流。
-        // 当 Device 状态发生变化或者 Device 消失时，ListAndWatch
-        // 会返回新的列表。
-        rpc ListAndWatch(Empty) returns (stream ListAndWatchResponse) {}
-
-        // Allocate 在容器创建期间调用，这样设备插件可以运行一些特定于设备的操作，
-        // 并告诉 kubelet 如何令 Device 可在容器中访问的所需执行的具体步骤
-        rpc Allocate(AllocateRequest) returns (AllocateResponse) {}
-
-        // GetPreferredAllocation 从一组可用的设备中返回一些优选的设备用来分配，
-        // 所返回的优选分配结果不一定会是设备管理器的最终分配方案。
-        // 此接口的设计仅是为了让设备管理器能够在可能的情况下做出更有意义的决定。
-        rpc GetPreferredAllocation(PreferredAllocationRequest) returns (PreferredAllocationResponse) {}
-
-        // PreStartContainer 在设备插件注册阶段根据需要被调用，调用发生在容器启动之前。
-        // 在将设备提供给容器使用之前，设备插件可以运行一些诸如重置设备之类的特定于
-        // 具体设备的操作，
-        rpc PreStartContainer(PreStartContainerRequest) returns (PreStartContainerResponse) {}
-  }
-  ```
-
-  {{< note >}}
-  <!--
-  Plugins are not required to provide useful implementations for
-  `GetPreferredAllocation()` or `PreStartContainer()`. Flags indicating which
-  (if any) of these calls are available should be set in the `DevicePluginOptions`
-  message sent back by a call to `GetDevicePluginOptions()`. The `kubelet` will
-  always call `GetDevicePluginOptions()` to see which optional functions are
-  available, before calling any of them directly.
-  -->
-  插件并非必须为 `GetPreferredAllocation()` 或 `PreStartContainer()` 提供有用的实现逻辑，
-  调用 `GetDevicePluginOptions()` 时所返回的 `DevicePluginOptions`
-  消息中应该设置这些调用是否可用。`kubelet` 在真正调用这些函数之前，总会调用
-  `GetDevicePluginOptions()` 来查看是否存在这些可选的函数。
-  {{< /note >}}
-
-<!--
-* The plugin registers itself with the kubelet through the Unix socket at host
-  path `/var/lib/kubelet/device-plugins/kubelet.sock`.
-
-* After successfully registering itself, the device plugin runs in serving mode, during which it keeps
-  monitoring device health and reports back to the kubelet upon any device state changes.
-  It is also responsible for serving `Allocate` gRPC requests. During `Allocate`, the device plugin may
-  do device-specific preparation; for example, GPU cleanup or QRNG initialization.
-  If the operations succeed, the device plugin returns an `AllocateResponse` that contains container
-  runtime configurations for accessing the allocated devices. The kubelet passes this information
-  to the container runtime.
--->
-* 插件通过 Unix socket 在主机路径 `/var/lib/kubelet/device-plugins/kubelet.sock`
-  处向 kubelet 注册自身。
-* 成功注册自身后，设备插件将以服务模式运行，在此期间，它将持续监控设备运行状况，
-  并在设备状态发生任何变化时向 kubelet 报告。它还负责响应 `Allocate` gRPC 请求。
-  在 `Allocate` 期间，设备插件可能还会做一些设备特定的准备；例如 GPU 清理或 QRNG 初始化。
-  如果操作成功，则设备插件将返回 `AllocateResponse`，其中包含用于访问被分配的设备容器运行时的配置。
-  kubelet 将此信息传递到容器运行时。
+1. 初始化。在这个阶段，设备插件将执行供应商特定的初始化和设置，以确保设备处于就绪状态。
+
+2. 插件使用主机路径 `/var/lib/kubelet/device-plugins/` 下的 Unix 套接字启动一个
```

Comment:
I think Unix is more common (though we know UNIX is more correct), and the English version use Unix instead of UNIX.

I would personally prefer Unix, maybe we can open a discussion issue or let the PR author decide?